prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -202,7 +214,13 @@ for det in `echo $LIST_OF_DETECTORS | sed "s/,/ /g"`; do
     has_detector_reco $det && add_comma_separated TRACK_SOURCES "$det"
   fi
 done
-PVERTEX_CONFIG="--vertexing-sources $TRACK_SOURCES --vertex-track-matching-sources $TRACK_SOURCES"
+
+if [ -z $ISPILOTDATA ]; then
+  PVERTEX_CONFIG="--vertexing-sources $TRACK_SOURCES --vertex-track-matching-sources $TRACK_SOURCES"
+else
+  PVERTEX_CONFIG="--vertexing-sources ITS --vertex-track-matching-sources ITS --hbfutils-config none"
+  PVERTEX_EXTRA_CONFIG="pvertexer.nSigmaTimeCut=100;pvertexer.dbscanMaxDist2=30;pvertexer.dcaTolerance=3.;pvertexer.pullIniCut=100;pvertexer.addZSigma2=0.1;pvertexer.tukey=20.;pvertexer.addZSigma2Debris=0.01;pvertexer.addTimeSigma2Debris=1.;pvertexer.maxChi2Mean=30;"
+fi
```

Comment:
this too can be set via env.vars.

Reply:
Would setting it only if the env variables are not already set be ok for you? I would keep it in the script, to have less env variables when running.

Reply:
see above.

Reply:
As I mentioned above, I would give priority to what comes from outside, but also have some defaults in case nothing is provided. 
A sort of side comment: from past experience having settings only on the grid is not always easy to trace afterwards.

Reply:
In order to allow that `PVERTEX_CONFIG` can be set from the outside we need to change the way it is defined in the dpl-workflow.sh script, so we should at least change this to something like
```
if [ -z $PVERTEX_CONFIG ]; then
  PVERTEC_CONFIG=...
fi
```

Reply:
Sorry, commented below before I saw @shahor02 comment here. I agree I would not hardcode stuff in the general script. I think this should come via env vars. To make it clean, I would propose to collect all env vars you set in an extra script. If you want to track which extra env vars you set in the logs (to avoid confusion when running something on the grid later), that extra script could have some logic to print the extra env vars it is setting. Then you would always have it logged.
@martenole : see my comment below about adding `VERTEXING_SOURCES`

Reply:
I would anyway always print everything. Experience showed that it is very useful. 
@martenole : yes, your comment is in line to what I wrote earlier to Ruben ("Would setting it only if the env variables are not already set be ok for you?"), but let's change approach.

Reply:
Done.